yaml/roads.yaml

```diff
@@ -31,6 +31,7 @@ global:
       leisure: {col: leisure}
       sport: {col: sport}
       man_made: {col: man_made}
+      walking_network: {expr: "mz_hiking_network(osm_id)"}
```

Comment:
For #647, we'd add something like,

* `bicycle_network: {expr: "mz_cyling_network(osm_id)"}`